### ISSUE TITLE
Indicate post-publish edits

### ISSUE THREAD

**Issue by PhilMunk** [jsecretan/maven-public#5]
When a post or comment gets edited by the author, the timestamp should say something like "last edited 5 minutes ago".

This would give readers a better chance to understanding the situation, when someone edits their post or comment in response to someone else's comment.

**Comment by genidma** [comment]
Yes. This is a good feature. Also, if someone is trolling a user (Post a question/comment and then changing it upon receiving answers/updates), then there will be a trail/history.
